subworkflows/local/vcf_impute_glimpse2/main.nf

```diff
@@ -0,0 +1,46 @@
+include { GLIMPSE2_PHASE                 } from '../../../modules/nf-core/glimpse2/phase/main'
+include { GLIMPSE2_LIGATE                } from '../../../modules/nf-core/glimpse2/ligate/main'
+include { BCFTOOLS_INDEX as INDEX_PHASE  } from '../../../modules/nf-core/bcftools/index/main.nf'
+include { BCFTOOLS_INDEX as INDEX_LIGATE } from '../../../modules/nf-core/bcftools/index/main.nf'
+
+workflow VCF_IMPUTE_GLIMPSE2 {
+
+    take:
+    ch_input        // channel (mandatory): [ meta, vcf, csi, infos ]
+    ch_panel        // channel (mandatory): [ meta, vcf, csi, region ]
```

Comment:
Shouldn't the panel be a `.bin` file ?

Reply:
`Glimpse2 `can handle the imputation with the `region1` and `region2` and also with the `bin` files. If the pipeline comes from the `panelprep` step, then it should use the `region1` and `region2` channels. If it starts from the `impute` step, it would be more efficient to start from the `bin` files.

Reply:
Okay, so we might need to prepare for both configurations.